### ISSUE TITLE
Config to pull pyright

### ISSUE THREAD

**Issue by KotlinIsland** [DetachHead/basedpyright#233]
When pyright does a release, we create a PR, with a separate action to merge it into main. The only complication is that merge PR's are banned.

**Comment by DetachHead** [comment]
would it make a PR? that would be annoying because merge PRs can't actually be merged through github, so we would need to also make an action that does the merge as well

**Comment by KotlinIsland** [comment]
Yeah, something like that, or maybe it can override the rules about merge PR via the api?

**Comment by DetachHead** [comment]
this is always going to need to be a manual process, i dont think theres any point attempting to automate any of it
